schema/tables/process_file_events.yml

```diff
@@ -0,0 +1,146 @@
+name: process_file_events
```

Comment:
Missing
```
platforms: 
  - linux
```
And:
```
evented: true
```

Reply:
That's what I though too but it looks like we don't need these. See the `process_events` table on the website here: https://fleetdm.com/tables/process_events:

![Screenshot 2024-02-01 at 10 01 23 AM](https://github.com/fleetdm/fleet/assets/47070608/c723947d-35c0-4381-b426-ba174cee0411)

`process_events` shows up w/ "Evented" indicator and has the correct platforms even though they're not specified in the override file here: https://github.com/fleetdm/fleet/blob/main/schema/tables/process_events.yml

I'm assuming this is because we're adding this info some other way.

@eashaw what's the convention here? Should we be specifying `platforms` and `evented` (optional fields) in the overrides file?

Reply:
Ah makes sense! (There's seems to be some merging of information happening behind the scenes.)

Reply:
>What's the convention here? Should we be specifying platforms and evented (optional fields) in the overrides file?

We typically only include the values that we want to change in the osquery schema in our override files. The optional values are only required if the table is being added to the schema by the YAML file.